app/src/main/java/org/dhis2/data/service/SyncPresenterImpl.kt

```diff
@@ -402,13 +402,13 @@ class SyncPresenterImpl(
         val enrollment = d2.enrollmentModule().enrollments().uid(uid).blockingGet()
         Completable.fromObservable(
             d2.trackedEntityModule().trackedEntityInstances()
-                .byUid().eq(enrollment.trackedEntityInstance()!!)
-                .byProgramUids(listOf(enrollment.program()!!))
+                .byUid().eq(enrollment?.trackedEntityInstance())
```

Comment:
can this lead to an exception? `byUid().eq(enrollment?.trackedEntityInstance())` if enrollment is null?